bundles/binding/org.openhab.binding.modbus/src/main/java/net/wimpi/modbus/net/SerialConnection.java

```diff
@@ -102,7 +102,7 @@ public void open() throws Exception {
           CommPortIdentifier.getPortIdentifier(m_Parameters.getPortName());
     } catch (NoSuchPortException e) {
       if(Modbus.debug) System.out.println(e.getMessage());
```

Comment:
Agreed :-) Would be great if you could send a new PR for that.

Enjoy your holidays!

Thanks, Thomas E.-E.